Gruntfile.js

```diff
@@ -79,7 +79,7 @@ module.exports = function( grunt ) {
 			},
 			test: {
 				src: [ "test/*.js", "test/functional/**/*.js", "test/unit/**/*.js",
-					"!test/config.js" ],
+					"test/compiler/**/*.js", "!test/config.js", "!test/compiler/_compiled/**" ],
```

Comment:
It also pollutes your working directory. Instead of having a single `test` directory, you'll now have `test`, `test-compiler` and `test-compiler-generated`.